prerender-server/client

```diff
@@ -1 +1 @@
-../client/src
```

Comment:
why this change?

Reply:
We're running against the source code instead of the compiled code now:

* The prerender-server imports from `../client/run-server`
* The client code is written in ES6+ and needs Babel transpilation
* build.sh runs Babel to transpile `client/src` → `client/dist`
* The symlink ensures the `prerender-server` uses the transpiled/built code, not the raw source